### DISCUSSION TITLE
fan control via configable ec register writing

### DISCUSSION THREAD

**Discussion by tkkcc** [Rem0o/FanControl.Releases#2542] [Q&A]
i am able to control fan speed via RWeveryting or ec_probe from nbfc, but nbfc self does not work for my device (https://github.com/UraniumDonut/nbfc-revive/issues/27). I wonder if FanControl can be used to write EC register based on temperature

**Comment by Rem0o** [comment]
It sure could. You can mess arround with a [plugin](https://github.com/Rem0o/FanControl.Releases/wiki/Plugins) to inject a control in the software, with the underlying ec register writing logic in it.
